content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,85 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, which will be released NEXT WEEK, a new major version of our free open source DJ software. It took about two years since the last release due to the large number of new features and changes under the hood.
+
+Here you can find some highlights of the Mixxx 2.4 features:
+
+### EngineOS export
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine DJ OS devices (Denon and Numark standalone controllers).
+This allows users to prepare their tracks on a laptop using Mixxx, export them on an USB stick and plug it to the standalone controller at your gig.
+
+### Saved Loops
+Mixxx 2.4 adds support for Saved Loops
+If you have already tracks with Saved Loops defined by Serato DJ, Mixxx 2.4 will automatically read them from the track metadata.
+
+<video autoplay loop><source src="/images/news/SavedLoopsFeature.mp4" type='video/mp4'></video>
```

Comment:
Thanks for making a video demo.

Instead of committing a 4 MB file into this git repo (which whill slow down cloning the repo without git-lfs), we should just upload it to the official Mixxx YouTube channel and use the YouTube embed mechanism (see below). This also allows us to get additional exposure on YouTube.

```suggestion
@Video(https://www.youtube.com/watch?v=INSERT_VIDEO_ID)
```

Reply:
I fear than we need to add this site to the Mixxx Privacy Policy.

Reply:
Quick fix: screenshot linked to youtube who have their own GDPR dialog?

Else, we don't need to reinvent the wheel, there are many "youtube embed gdpr" code snippets that fetch the image, add the warning.

Reply:
If you embed it like this, users already need to accept the YouTube terms before they can watch the video.

Example: https://mixxx.org/news/2020-08-22-portable-mixxx-setup/

Reply:
okay, great

Reply:
Who has access to the "official Mixxx YouTube channel" ?

Reply:
I do

Reply:
(reminder: video needs to be updated once it's finalized https://mixxx.zulipchat.com/#narrow/stream/245858-promotion/topic/2.2E4.20Feature.20Video.3A.20EngineOS.20export)